package.json

```diff
@@ -48,16 +48,16 @@
   },
   "dependencies": {
     "@restart/hooks": "^0.3.1",
```

Comment:
it seens to works for me with 15.0... 
maybe we can also bump other deps like @restart/hooks 👍